src/lem/multiframe.rs

```diff
@@ -406,25 +405,12 @@ impl<'a, F: LurkField, C: Coprocessor<F> + 'a> MultiFrameTrait<'a, F, C> for Mul
         store: &Self::Store,
         limit: usize,
         lang: &Lang<F, C>,
-    ) -> std::result::Result<Vec<Self::EvalFrame>, ProofError> {
+    ) -> Result<Vec<Self::EvalFrame>, ProofError> {
         // TODO: integrate https://github.com/lurk-lab/lurk-rs/commit/963a6361701efcaa78735d8fc9c927f518d8e31c
```

Comment:
You may want to clean up the TODO as well?

Reply:
I'm not sure. This function has deeper issues with its API: it only works for IVC.

LEM's infra doesn't use it (except in the proving tests) and we might just nuke it or make deeper changes after we sunset the Alpha infra.

Reply:
Then we should perhaps clarify the wording of the TODO? My point is here is a comment that recommends using a function, which is not edited in the commit that, precisely, uses that function.

If you want to formulate forward-looking guidance on what's left to be done with the `MultiFrame`'s logic for getting evaluation frames, that's great. Let's just make sure it reflects our thinking in more eloquent ways.

Reply:
Ah, I just realized what you meant by "clean up". It's done 👍🏼